be/src/storage/compaction_manager.h

```diff
@@ -128,6 +135,8 @@ class CompactionManager {
 
     int get_waiting_task_num();
 
+    ThreadPool* get_compaction_thread_pool() { return _compaction_pool.get(); }
```

Comment:
```suggestion
    ThreadPool* TEST_get_compaction_thread_pool() { return _compaction_pool.get(); }
```

This interface is added just for TEST purpose.

or more specifically
```
int32_t TEST_get_compaction_thread_pool_max_threads() const { return _compaction_pool->max_threads(); }
```